lib/Makefile.am

```diff
@@ -68,7 +68,7 @@ pkginclude_HEADERS = \
 	command_graph.h \
 	command.h \
 	memory.h network.h prefix.h routemap.h distribute.h sockunion.h \
-	stream.h table.h thread.h vector.h version.h vty.h zebra.h \
```

Comment:
The file needs to be listed here so it is installed to `/usr/include/frr`, which it should be in order to have a good reference to the FRR version in the installed headers.